infra/resources/src/main/res/values/dimens.xml

```diff
@@ -25,6 +25,7 @@
 
     <!-- region Text size -->
     <dimen name="text_size_large">16sp</dimen>
+    <dimen name="text_size_x_large">20sp</dimen>
```

Comment:
We should not be using the text sizes directly, only via text styles. I am pretty sure that large and title sizes are not used anywhere and should have been deleted at some point.